inference/inference.py

```diff
@@ -35,14 +37,20 @@ def main(
     enable_saleforce_content_safety: bool=True, # Enable safety check woth Saleforce safety flan t5
     **kwargs
 ):
-    if prompt_file is not None:
+    print (f'prompt file {prompt_file}')
+    if prompt_file and output_file:
```

Comment:
should we still allow reading a prompt file when the user doesn't specify an output file?

if not, the error message will be confusing when a user specifies a prompt_file but not an output_file and gets "No user prompt provided"